apis/quay/v1/quayregistry_types.go

```diff
@@ -752,6 +754,40 @@ func GetEnvOverrideForComponent(quay *QuayRegistry, kind ComponentKind) []corev1
 	return nil
 }
 
+// GetLabelsOverrideForComponent returns overriden labels for the provided component
+// nil is returned if there are no label overrides
+func GetLabelsOverrideForComponent(quay *QuayRegistry, kind ComponentKind) map[string]string {
```

Comment:
The sad story here is that as the config-tool is also part of the "quay" component the same labels will apply for both of them :-(